information_system/information_system/urls.py

```diff
@@ -29,10 +29,10 @@ class UserViewSet(viewsets.ModelViewSet):
     path('api/v1/persons/<int:pk>/', PersonAPIDetailView.as_view()),
     path('api/v1/employees/', EmployeeAPIView.as_view()),
     path('api/v1/employees/<int:pk>/', EmployeeAPIDetailView.as_view()),
+    path('api/v1/employees/<int:pk>/vacations/', EmployeeVacationAPIView.as_view()),
     path('api/v1/positions/', PositionAPIView.as_view()),
     path('api/v1/positions/<int:pk>/', PositionAPIDetailView.as_view()),
     path('api/v1/departments/', DepartmentAPIView.as_view()),
     path('api/v1/departments/<int:pk>/', DepartmentAPIDetailView.as_view()),
-    path('api/v1/historyVacations/', HistoryVacationAPIView.as_view()),
-    path('api/v1/historyVacations/<int:pk>/', HistoryVacationAPIDetailView.as_view()),
+    path('api/v1/history-vacations/', HistoryVacationAPIView.as_view()),
```

Comment:
It seems this end-point is redundant since it shows all vacations for all users, which is not informative enough.